MAVProxy/modules/mavproxy_silvus.py

```diff
@@ -201,21 +198,23 @@ def get_radio_data(self):
         if len(remoteip.split('.')) != 4:
             return
 
-        localnode = str(self.ip_to_id(localip))
-        remotenode = str(self.ip_to_id(remoteip))
+        localnode = str(self.silvus_settings.gnd_node)
+        remotenode = str(self.silvus_settings.air_node)
```

Comment:
So we stop deriving one from the other and get the user to set the node ID manually.

I guess this allows for compatability with both versions of the streamcaster firmware, but might have been nice to have the newer octets-to-nodeid conversion for a node if the node ID was zero in the settings.

Reply:
the reason for the change is each radio has a 172.x.x.x address and a 192.y.y.y address. The node ID can only be derived from the 172 (native) address, but we commonly only have a route to the 192 ("private") address